### ISSUE TITLE
Example of setting org-now-location

### ISSUE THREAD

**Issue by oatmealm** [alphapapa/org-now#7]
Can you give an example of how to set up? I've currently got this, but it doesn't seem to be working correctly.
Is `org-now-location` a file or a heading in the current document I'm working on? Do i need to set it up manuall every time before I start using `org-now`?

```
(use-package! org-now
  :config
  (setq! org-now-location (concat (expand-file-name "now.org" xxx/org) "::*Work"))
  (map! :map org-mode-map
        :localleader
        :prefix "N"
        :n "rl" #'org-now-link
        :n "rn" #'org-now-refile-to-now
        :n "rp" #'org-now-refile-to-previous-location))
```

**Comment by alphapapa** [comment]
If you use the customization system (e.g. `M-x customize-option RET org-now-location RET`), you will be guided to the use the correct format.  As well, the variable's docstring explains it.  Emacs is the self-documenting editor.  ;)

**Comment by oatmealm** [comment]
I'm aware of what Emacs is, yet the instructions provided on the customization screen are really not clear at all. Why would you want people that use your module to guess?

**Comment by alphapapa** [comment]
As far as I'm aware, that variable's docstring is valid English that explains precisely what the variable's format is.

> Why would you want people that use your module to guess?

Why would you say something like that?  Do you realize how rude that sounds?  Do you have any idea how many hours I've spent on these Org packages, writing extensive documentation in the code, docstrings, readme files, and info manuals?  And then you say that I want people to guess?

**Comment by oatmealm** [comment]
I've asked a simple question and got an arrogant response. If you're not interested or too busy it's better to ignore. I know how to look up help in Emacs the text is by no means obvious, unless you're familiar with the particular jargon. I've only now realized what you mean, trying different formats and it's not working for me. Thanks for your hard work but I think I'll move on. I've got better things to do then solve riddles.
